utils/scapp.h

```diff
@@ -455,6 +455,7 @@ class scApp : public ktxApp {
         int          astc;
         ktx_bool_t   normalMode;
         ktx_bool_t   normalize;
+        ktx_bool_t   perceptual;
```

Comment:
perceptual should be in `astcOptions` as it is not valid for the basisU encoder.